res/controllers/Denon Prime 4.midi.xml

```diff
@@ -0,0 +1,2394 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.4.0+">
+    <info>
+        <name>Denon Prime 4 - Mixxx Mapping</name>
+        <author>Levi "Whanake" Williams</author>
+        <description>This is a Work-In-Progress mapping to allow the Denon Prime 4 to be used with Mixxx, ideally with the same (if not more) functionality that the Prime 4 provides as a standalone device.</description>
+        <wiki>https://github.com/mixxxdj/mixxx/wiki/Denon-Prime-4</wiki>
+        <forum>https://mixxx.discourse.group/t/denon-prime-4-mapping/22404</forum>
```

Comment:
Please keep in mind that we also need a manual page so we can include this mapping. The wiki has a section on that:
https://github.com/mixxxdj/mixxx/wiki/Contributing-Mappings#documenting-the-mapping

Reply:
I'll get on that ASAP